pipeline/run_benchmark.sh

```diff
@@ -3,7 +3,7 @@
 #TODO: parameter validation
 
 set -o pipefail
-# set -x
+set -x
```

Comment:
You added this for debugging only, right?  This will make regular deployments too verbose.